apps/gallery/locales/gallery.en-US.properties

```diff
@@ -100,11 +100,14 @@ back-button.ariaLabel = Back
 
 # Edit crop buttons
 crop-options.ariaLabel = Crop
-crop-revert.ariaLabel = Revert
 crop-aspect-free.ariaLabel = Custom aspect ratio
 crop-aspect-portrait.ariaLabel = Portrait aspect ratio
 crop-aspect-landscape.ariaLabel = Landscape aspect ratio
 crop-aspect-square.ariaLabel = Square aspect ratio
+crop-aspect-more.ariaLabel = More aspect ratios
+crop-aspect-3x4.ariaLabel = 3:4
+crop-aspect-4x3.ariaLabel = 4:3
+crop-aspect-16x9.ariaLabel = 16:9
```

Comment:
rop-aspect-3x4.ariaLabel = 3:4 is setting both the text and ariaLabel of the menu item. I like the suggestion for both the strings and the aria labels to say "3:4 portrait" and "4:3 landscape" and "16:9 landscape" . 

If we need to make aria label even more verbose then we should use separate strings.

I believe you are right, string freeze is usually in effect after FL date, https://wiki.mozilla.org/Release_Management/B2G_Landing